app/src/main/java/com/stl/skipthelibrary/BookDatabaseHelper.java

```diff
@@ -0,0 +1,82 @@
+package com.stl.skipthelibrary;
+
+import android.net.Uri;
+import android.util.Log;
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class BookDatabaseHelper {
+    private static final String TAG = "BookDatabaseHelper";
+    private static final String BASE_URL = "https://www.googleapis.com/books/v1/volumes?";
+    private static final String QUERY_PARAM = "q";
+    private static final String QUERY_BASE = "=isbn:";
+    private static final String PRINT_TYPE = "printType";
+    private static final String PRINT_VALUE = "books";
+    private static final String METHOD = "GET";
+
+    public BookDatabaseHelper() {}
```

Comment:
Perhaps we should rename the DatabaseHelper class to UserDatabaseHelper for continuity?